src/Presentation/Camera/Fetching/Types/TPCamera.cs

```diff
@@ -11,8 +11,13 @@ public class TPCamera : InputObjectGraphType<PCamera>
     {
         public TPCamera()
         {
-            Field(camera => camera.Ids, true).Description("This is the list of ids");
-            Field(camera => camera.Pagination, false, typeof(TPPagination)).Description("This is the pagination.");
+            Name = "CameraParameter";
+            Description = "Cameras will be filtered by this fields";
+
+            Field(camera => camera.Ids, true).Description("If no value will be given, then the cameras will not be filtered by id.");
```

Comment:
Should we move the description to the next line and also a new line between the fields? (for all the create types)

Reply:
done